github/login_test.go

```diff
@@ -108,3 +109,23 @@ func TestValidateResponse(t *testing.T) {
 	assert.Equal(t, ErrUnableToGetGithubUser, validateResponse(validUser, invalidResponse, nil))
 	assert.Equal(t, ErrUnableToGetGithubUser, validateResponse(&github.User{}, validResponse, nil))
 }
+
+func Test_githubClientFromAuthURL(t *testing.T) {
+	for _, test := range []struct {
```

Comment:
While this is valid, can you define cases separately? (kinda a nitpick, but it matches other libraries)

```
cases := []struct{
    ...
}{
    ...
}
for _, c := range cases {
   ...
}
```